docs/application/vstools/tools/dotnet-diagnostics.md

```diff
@@ -0,0 +1,117 @@
+    Now click the **Collect** button provided in **Diagnostics GCDump Explorer** window.
+
+     ![Run dotnet-gcdump](media/dotnet-gcdump.png)
+
+
+   - If everything is ok then the application starts and it starts collecting the gcdump file.
+
+     ![Tizen application running](media/dotnet-gcdump-collect.png)
+
+
+   - Once the gcdump file is collected, a message popup is shown to inform the user, and the file is added to the gcdump file list in **GCDump Explorer window**.
+
+
+   - User can collect the gcdump file multiple times by clicking the **Collect** button, until the application is closed by pressing "Home" or "Back" buttons, or by clicking the **Stop** button given in **GCDump Explorer window**.
+
+
+
+### Step 2. Analyze the gcdump file
+   - The GUI application provides two tools to view the gcdump diagnostics data. The views include:
+
+       - Double click the gcdump file in **GCDump Explorer window**. It will open the file in **PerfView** tool.
+       ![gcdump PerfView analysis view](media/dotnet-gcdump-perfview.png)
+       - Right click the gcdump file in **GCDump Explorer window** and select **Open With Editor**. It will open the file in **Visual Studio**.
+
+
+## dotnet-trace
+
+### Step 1. Run dotnet-trace tool
+   - If no Tizen device is connected and no Tizen emulators are running then launch the type of emulator you want to use for diagnosing your application.
+
+     ![Launch Emulator](media/start_emulator.png)
+
+   - In the menu, choose **Tools > Tizen > .NET Core Diagnostics > dotnet-trace**.   
+    Now select the trace **Profile**, **Format** and **Duration** options and click the **Collect** button provided in **Diagnostics Trace Explorer** window.
+
+     ![Run dotnet-dump](media/dotnet-trace.png)
+
+
+   - If everything is ok then the application starts and it starts collecting the trace.  
+       - If application is not running already in the device/emulator then **trace** will be collected from application startup.
+       - If application is already running in the device/emulator then **trace** will be collected from there on.
+
+     ![Tizen application running](media/dotnet-trace-collect.png)
+
+   - Press **Stop Trace** button above to stop and generate the trace file.
+
+   - Once the trace file is generated, a message popup is shown to inform the user, and the file is added to the gcdump file list in **Dump Explorer window**.
+
+
+
+### Step 2. Analyze the trace file
+   - The GUI application provides different tools for different **Format** to view the trace diagnostics data.
+   - **nettrace**
+       - Double click the trace (.nettrace) file in **Trace Explorer window**. It will open the file in **PerfView** tool.
+       ![trace PerfView analysis view](media/dotnet-trace-perfview.png)
+       - Right click the trace file in **Trace Explorer window** and select **Open With Editor**. It will open the file in **Visual Studio**.
```

Comment:
```suggestion
       - Right click the trace file in **Trace Explorer** window and select **Open With Editor**. It will open the file in **Visual Studio**.
```